Source/Other/Extensions/UserDefaultsExtension.swift

```diff
@@ -28,4 +28,9 @@ extension UserDefaults {
 		}
 		return savedFont
 	}
+
+    @objc dynamic var SPSecureBookmarks: [Dictionary<String, Data>] {
```

Comment:
Is the dynamic necessary?
And what about the fallback value? :)

Reply:
Is the dynamic necessary? - no, I initially thought I would do some KVo from Objc, but I'm not.
And what about the fallback value? - best I could think of. Can't be nil.

I check from this later for empty pref:

```swift
// FIXME: check for default empty value - I don't like this.
if bookmarks.isEmpty == true || bookmarks[0]["noBookmarks"] != nil {
  os_log("Could not get secureBookmarks from prefs.", log: self.log, type: .error)
  Crashlytics.crashlytics().log("Could not get secureBookmarks from prefs.")
  bookmarks.removeAll()
  return
}
```

Reply:
Yeah I commented there as well :)